dev/firewalld_integration_test.sh

```
@@ -102,15 +98,15 @@ test_knocker_zone_creation() {
     
     # Check zone properties (verify default DROP rules exist for monitored ports)
     # We no longer rely on the zone target; instead ensure monitored ports have DROP rules
-    zone_info=$(docker compose exec -T knocker firewall-cmd --zone=knocker --list-all || true)
+    zone_info=$(docker compose exec -T knocker firewall-cmd --zone=knocker --list-all)
```

Shellcheck (0.11.0)

[warning] zone_info appears unused. Verify use (or export if used externally).

(SC2034)